deploy/operator/base/rbac.yaml

```diff
@@ -179,6 +187,10 @@ metadata:
   labels:
     rbac.authorization.k8s.io/aggregate-to-edit: "true"
 rules:
+  - apiGroups: ["security.openshift.io"]
```

Comment:
I think this is a remnant of my attempt to fix the artifact cache perm issue

should not be causing current issues I'm seeing in this PR

Reply:
This appears to be the only thing that could actually be affecting the cache, although I don't see how it would affect it.

Reply:
unable to test locally (still in kcp debug) but I removed this remnant just to eliminate it as a variable 

let's see what CI says

Reply:
sure enough, the e2e is passing again

openshift must have been performing some voodoo under the covers with that partial application of anyuid, but exactly what, who knows .... a mystery unfortunately I just do not have time to dive into